internal/controller/http/v1/get_entities.go

```diff
@@ -0,0 +1,36 @@
+package v1
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/kordape/ottct-main-service/internal/handler"
+	"github.com/kordape/ottct-main-service/pkg/api"
+)
+
+func (r *routes) getEntitiesHandler(entityManager handler.EntityManager) func(c *gin.Context) {
+	return func(c *gin.Context) {
+		r.l.Debug("Get entities request received")
+
+		entities, err := entityManager.GetEntities()
+		if err != nil {
+			r.l.Error(fmt.Errorf("GetEntities internal error: %w", err))
+			c.AbortWithStatusJSON(http.StatusInternalServerError, err)
+			return
+		}
+
+		// TODO posto je handler.Entity : api.Entity mapiranje 1:1 da li da radimo ovaj blok?
```

Comment:
Tako se ispostavilo da je mapiranje 1:1 ali ne mora da znaci da ce tako i ostati, pa je dobro da razdvojimo na ta dva layera. Mislim to je best practice da svaki layer radi sa svojim modelom i medjusobno ih konvertuju. Tako da ja bih ostavio

Reply:
Vazi se 😊